Units/ocaml_empty.ml_xref.t/expected.tags-x

```diff
@@ -1 +0,0 @@
-Input            module        1 input.ml 
```

Comment:
Semantic uses the direct "tags" format from stdout. It builds commands and executes them in a subshell. The commands look something like `/usr/bin/ctags --language-force=ocaml --sort=no --excmd=number --fields=aKStsim --ocaml-kinds=cmMvptfCre -f - ~/code/ocaml/test.ml`